### ISSUE TITLE
Mod crashes on M1 Mac / Warden

### ISSUE THREAD

**Issue by xnousnow** [gXLg/libr-getter#22]
The mod crashes when I start the mod.
Prism Launcher / Apple M1, MacBook Air / Fabric 1.20.2 / Sodium, Indium, Librarian Getter Installed / On server

```
[19:31:21] [Render thread/INFO]: [System] [CHAT] Goals list:\n- minecraft:feather_falling 4 (24) (remove)
[19:31:25] [Render thread/INFO]: [System] [CHAT] The lectern is not been set!
[19:31:30] [Render thread/INFO]: [System] [CHAT] Villager selected
[19:31:32] [Render thread/INFO]: [System] [CHAT] Block selected
[19:31:34] [Render thread/INFO]: [System] [CHAT] LibrGetter process started
[19:31:35] [Render thread/ERROR]: Unreported exception thrown!
java.lang.NoSuchMethodError: 'void net.minecraft.class_634.method_2883(net.minecraft.class_2596)'
	at com.gxlg.librgetter.Worker.tick(Worker.java:231) ~[libr-getter%201.3.3%20Warden.jar:?]
	at net.minecraft.class_636.handler$zlf000$librgetter$tick(class_636.java:1602) ~[client-intermediary.jar:?]
	at net.minecraft.class_636.method_2927(class_636.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java:1905) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1237) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:856) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:253) ~[minecraft-1.20.2-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.3.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) ~[NewLaunch.jar:?]
[19:31:35] [Sound engine/ERROR]: Stop: Invalid name parameter.
[19:31:35] [Render thread/INFO]: Stopping worker threads
---- Minecraft Crash Report ----
// I feel sad now :(
Time: 2024-01-19 19:31:35
Description: Unexpected error
java.lang.NoSuchMethodError: 'void net.minecraft.class_634.method_2883(net.minecraft.class_2596)'
	at com.gxlg.librgetter.Worker.tick(Worker.java:231)
	at net.minecraft.class_636.handler$zlf000$librgetter$tick(class_636.java:1602)
	at net.minecraft.class_636.method_2927(class_636.java)
	at net.minecraft.class_310.method_1574(class_310.java:1905)
	at net.minecraft.class_310.method_1523(class_310.java:1237)
	at net.minecraft.class_310.method_1514(class_310.java:856)
	at net.minecraft.client.main.Main.main(Main.java:253)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------
-- Head --
Thread: Render thread
Stacktrace:
	at com.gxlg.librgetter.Worker.tick(Worker.java:231)
	at net.minecraft.class_636.handler$zlf000$librgetter$tick(class_636.java:1602)
	at net.minecraft.class_636.method_2927(class_636.java)
-- Affected level --
Details:
	All players: 1 total; [class_746['xn0u'/54112, l='ClientLevel', x=-1216.54, y=65.00, z=1572.20]]
	Chunk stats: 4096, 453
	Level dimension: minecraft:overworld
	Level spawn location: World: (20,95,-50), Section: (at 4,15,14 in 1,5,-4; chunk contains blocks 16,-64,-64 to 31,319,-49), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,-64,-512 to 511,319,-1)
	Level time: 79631000 game time, 23000660 day time
	Server brand: Spigot
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:451)
	at net.minecraft.class_310.method_1587(class_310.java:2481)
	at net.minecraft.class_310.method_1514(class_310.java:880)
	at net.minecraft.client.main.Main.main(Main.java:253)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
```

**Comment by gXLg** [comment]
This is what happens, when you don't test your releases. Man. I will fix tomorrow (in about 20 hours)

**Comment by xnousnow** [comment]
Also, for your information, the other 'Auto Lectern' mod crashes with same conditions.

**Comment by gXLg** [comment]
Hmm, after testing it seems to work for me. Also, what actually happens is probably some code changed from 1.20.1 to 1.20.2, which was adapted in the newer fabric api, but since my mod uses an older api version for compatibility - it crashes

**Comment by gXLg** [comment]
I don't really know from where this error is, but I will try to fix

**Comment by gXLg** [comment]
Fixed in [v1.3.4](https://github.com/gXLg/libr-getter/releases/tag/v1.3.4)
